trufflesuite/truffle
# Add `solc` as a devDep to compile-solidity-tests

## PR description

The *compiles w/ local path* test referenced a file in a node_modules folder that was already subject to moving due to lerna hoisting behavior. This uncertainty makes the test unreliable!

This PR adds a pinned version of solc as a devDep, ensuring the file will be available for testing purposes using `require.resolve`.

Note, we will have to update this pinned version as part of the bump solc chore.

## Testing instructions

check out the branch and run the *compiles w/ local path* test in `compile-solidity-tests`, or verify tests don't fail in CI

## Documentation


## Breaking changes and new features
